tokio/src/runtime/handle.rs

```diff
@@ -80,8 +81,9 @@ impl Handle {
     /// # Panic
     ///
     /// This will panic if called outside the context of a Tokio runtime. That means that you must
-    /// call this on one of the threads **being run by the runtime**. Calling this from within a
-    /// thread created by `std::thread::spawn` (for example) will cause a panic.
+    /// call this on one of the threads **being run by the runtime**, or from a thread with an active
+    /// `EnterGuard`. Calling this from within a thread created by `std::thread::spawn` (for example)
+    /// will cause a panic unless that thread has an active `EnterGuard`.
```

Comment:
Seems like this should also mention `spawn_blocking`? Also, can you reflow the text at 80 chars?

Reply:
Mention `spawn_blocking` as an alternative? Or do blocking threads also not have a runtime context?

Reply:
If you mention `Handle::enter` and the runtime worker threads as places that are _in_ the context, but do not mention `spawn_blocking` threads, then that seems like you are implying that the context is _not_ available in `spawn_blocking`. However, that is not the case: the context _is_ available in `spawn_blocking`.